databases/week3/MealReservation.sql

```diff
@@ -0,0 +1,138 @@
+INSERT INTO meal VALUES(2, 'pasta with chicken', 'with tomato and barbecue sauce', 'Spain', now(), 10, 100.00, '2020-08-08');
+INSERT INTO meal VALUES(3, 'lasagne vegetarian', 'with mushroom sauce', 'Italy', now(), 7, 90.00, '2020-08-05');
+INSERT INTO meal VALUES(4, 'pizza barbecue', 'thin crust', 'Denmark', now(), 15, 110.00, '2020-08-01');
+INSERT INTO meal VALUES(5, 'lasagne', 'with tomato sauce', 'Italy', now(), 5, 80.00, '2020-07-31');
+-- Get a meal with any id, fx 1
+SELECT * FROM meal WHERE id=1;
+-- Update a meal with any id, fx 1. Update any attribute fx the title or multiple attributes
+UPDATE meal SET title='lasagne', price=125.75 WHERE id=1;
+-- Delete a meal with any id, fx 1
+DELETE FROM meal WHERE id=1;
+
+-- RESERVATION
+-- Get all reservations
+SELECT * FROM reservation;
+-- Add a new reservation
+INSERT INTO reservation VALUES(1,5,1,current_date(),22223333,'Jaqueline','jaque_171@hotmail.com');
+INSERT INTO reservation VALUES(2,3,1,current_date(),11111111,'Marie','marie_hey@hotmail.com');
+INSERT INTO reservation VALUES(3,10,2,current_date(),33344455,'Martin','martin_007@hotmail.com');
+INSERT INTO reservation VALUES(4,5,2,current_date(),12312312,'Sonya','sonya_abc@hotmail.com');
+INSERT INTO reservation VALUES(5,7,3,current_date(),13131313,'Anna','anna_anna@hotmail.com');
+-- Get a reservation with any id, fx 1
+SELECT * FROM reservation WHERE id=1;
+-- Update a reservation with any id, fx 1. Update any attribute fx the title or multiple attributes
+UPDATE reservation SET contact_name='Jackie' WHERE id=1;
+-- Delete a reservation with any id, fx 1
+DELETE FROM reservation WHERE id=1;
+
+-- REVIEWS
+-- Get all reviews
+SELECT * FROM review;
+-- Add a new review
+INSERT INTO review VALUES(1, 'pasta-veg-Review', 'Just a opinion', 1, 2, current_date());
+INSERT INTO review VALUES(2, 'pasta-veg-Review', 'Just a opinion', 1, 5, current_date());
+INSERT INTO review VALUES(3, 'Lasagne-Review', 'Just a opinion', 5, 2, current_date());
+INSERT INTO review VALUES(4, 'Lasagne-Review', 'Just a opinion', 5, 4, current_date());
+INSERT INTO review VALUES(5, 'Lasagne-Review', 'Just a opinion', 5, 3, current_date());
+-- Get a review with any id, fx 1
+SELECT * FROM review WHERE id=1;
+-- Update a review with any id, fx 1. Update any attribute fx the title or multiple attributes
+UPDATE review SET description='just a personal opinion', stars=3 WHERE id=1;
+-- Delete a review with any id, fx 1
+DELETE FROM review WHERE id=1;
+
+-- ADDITIONAL QUERIES
+-- Get meals that has a price smaller than a specific price fx 90
+SELECT * FROM meal WHERE price<90;
+-- Get meals that still has available reservations
+SELECT 
+    meal.title Title,
+    meal.max_reservations Maximum_Reservations,
+    reservation.number_of_guests Number_of_Guests_Reserved
+FROM
+    meal
+        JOIN
+    reservation ON (meal.id = reservation.meal_id)
+WHERE
+    reservation.number_of_guests < meal.max_reservations;
```

Comment:
you need to get the sum of reservations for a specific meal for a specific date and compare it to the max_reservations.